src/languages/_english.json

```
@@ -46,7 +46,46 @@
         "testnet_label": "Your testnet transaction history starts now!"
       }
     },
+    "avatar_builder": {
+      "emoji_categories": {
+        "activities": "Activities",
+        "animals": "Animals & Nature",
+        "flags": "Flags",
+        "food": "Food & Drink",
+        "objects": "Objects",
+        "smileys": "Smileys & People",
+        "symbols": "Symbols",
+        "travel": "Travel & Places"
+      }
+    },
     "back_up": {
+      "cloud": {
+        "password": {
+          "backup_placeholder": "Backup Password",
+          "choose": "Choose a password",
+          "confirm_backup": "Confirm Backup",
+          "confirm_label": "Confirm password",
+          "confirm_placeholder": "Confirm Password",
+          "minimum_characters": "Minimum %{minimumLength} characters",
+          "not_matching": "Passwords don't match",
+          "not_recoverable": "It can't be recovered!",
```

Do you think you can follow this pattern?
```suggestion
          "it_cant_be_recovered": "It can't be recovered!",
```

---

@osdnk Yes, that makes sense. I'll go through and update the keys to try to match the text to make searching easier.